crates/era-cheatcodes/tests/src/cheatcodes/Persistent.t.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {Test, console2 as console} from "forge-std/Test.sol";
+import {Constants} from "./Constants.sol";
+
+contract Target {
+    function output() public pure returns (uint256) {
+        return 255;
+    }
+}
+
+contract DummyContract {
+    uint256 public val;
+
+    function hello() external view returns (string memory) {
+        return "hello";
+    }
+
+    function set(uint256 _val) public {
+        val = _val;
+    }
+}
+
+contract ForkPersistentTest is Test {
+    uint256 constant FORK_BLOCK = 19579636;
+    address constant DUMMY_ADDRESS = 0x1345df6d4C9c3035724fd0e3914dE96a5a83AAF4;
+
+    /// checks that marking as persistent works
+    function testMakePersistent() public {
+        uint256 mainnetFork = vm.createFork("mainnet", FORK_BLOCK + 100);
+        uint256 optimismFork = vm.createFork("mainnet", FORK_BLOCK + 100);
```

Comment:
so, we have two identical forks, is it on purpose? and why do you call one of them optimism?

Reply:
Maybe the names are misleading, I'll change them. It is similar to how foundry test MakePersistent.